### ISSUE TITLE
Beginner's Assumption #1: 1em === 16px

### ISSUE THREAD

**Issue by erickpatrick** [erickpatrick/erickpatrick.github.io#32]
```
---
title: Beginner's Assumption #1: 1em === 16px
date: 2019-09-10T00:00:00.000Z
description: >-
  Nowadays, starting to program usually means creating a simple web page using HTML and CSS. No JavaScript needed. However, how many things we take for granted that someone knows about but, as a beginner, she make assumptions...?
categories:
   - programming
featured: false
extends: _layouts.post
section: content
---
```

**Comment by erickpatrick** [comment]
The other day I was helping a friend on how to convert a JPEG mockup into HTML/CSS. I was not coding, but gave my inputs on how he could improve things here ant there. 

While we were discussing font sizes, he immediately started saying that a specific element had 16px of size. I started looking into his CSS code and could not find where was this specific rule and asked him where to find.

He was in shock, he thought I knew it already. I clearly stated I did not know where he added it and he said: "I had set to 1em, so It's the browser default value. How don't you know that?!" and then things started to get interesting and what I showed him blown his mind.

Basically, I asked him to go to his browser configuration and change the default font size to a huge number and to look again into his layout. He stared at it without believing his own eyes. Everything got huge.

After some time he asked why that happened and added that almost every publication he read had made calculations of 1em = 16px or didn't even bothered saying how many pixels it would be. Kindly, I said those publications were wrong and he should search a better sources and started explaining what was going on...
